homeassistant/components/zha/number.py

```diff
@@ -37,6 +38,7 @@
 CONFIG_DIAGNOSTIC_MATCH = functools.partial(
     ZHA_ENTITIES.config_diagnostic_match, Platform.NUMBER
 )
+MULTI_MATCH = functools.partial(ZHA_ENTITIES.multipass_match, Platform.NUMBER)
```

Comment:
not needed

Reply:
addressed with
https://github.com/home-assistant/core/pull/105765/commits/3435b63b82ce963cfc9c606cb9aadf91fe04a695